konf/site.yaml

```diff
@@ -11,9 +11,13 @@ env:
   - name: SENTRY_DSN
     value: https://6cacb25de947478faeb57f3836da1cf3@sentry.is.canonical.com//24
 
+  # # We have to use the staging APIs temporarily
+  # - name: CHARMSTORE_PUBLISHER_API_URL
+  #   value: https://api.staging.charmhub.io/
+
   # We have to use the staging APIs temporarily
-  - name: CHARMSTORE_PUBLISHER_API_URL
-    value: https://api.staging.charmhub.io/
+  - name: CHARMSTORE_API_URL
+    value: https://api.staging.snapcraft.io/
```

Comment:
@steverydz I think this is the issue, you don't need to remove `CHARMSTORE_PUBLISHER_API_URL`, you need to have both.

```sugestion
  - name: CHARMSTORE_API_URL
    value: https://api.staging.snapcraft.io/

  - name: CHARMSTORE_PUBLISHER_API_URL
    value: https://api.staging.charmhub.io/
```